xwayland/xwayland.c

```diff
@@ -405,3 +405,22 @@ void wlr_xwayland_set_seat(struct wlr_xwayland *xwayland,
 	xwayland->seat_destroy.notify = wlr_xwayland_handle_seat_destroy;
 	wl_signal_add(&seat->events.destroy, &xwayland->seat_destroy);
 }
+
+
+bool wlr_xwayland_surface_is_unmanaged(const struct wlr_xwayland_surface *surface) {
+	static enum atom_name needles[] = { NET_WM_WINDOW_TYPE_UTILITY,
```

Comment:
Style nitpick: add a newline after opening brackets and before closing brackets.